src/generated/cloud/discoveryengine/v1/.sidekick.toml

```diff
@@ -17,5 +17,6 @@ specification-source = 'google/cloud/discoveryengine/v1'
 service-config       = 'google/cloud/discoveryengine/v1/discoveryengine_v1.yaml'
 
 [codec]
+version        = '1.1.0'
```

Comment:
bad formatting, if you care.

Reply:
Ack.  Evidently we manually formatted this file once upon a time. I do not care that it is slightly off this time.